sdk/storage/azure-storage-file/src/samples/java/com/azure/storage/file/DirectorySample.java

```diff
@@ -4,12 +4,9 @@
 
 import com.azure.core.http.rest.Response;
 import com.azure.core.util.configuration.ConfigurationManager;
-import com.azure.storage.file.DirectoryClient;
```

Comment:
I am curious why these import useless. The DirectoryClient is still in use.

Reply:
It's because both files are in the same package.

Reply:
Classes that live in the same package do not have to be imported. They are done automatically. This file did not live in the same package name before.

Reply:
Thanks! What's the reason of making the sample under the same package?

Reply:
Jonathan mentioned that samples should live in the root package (ie. com.azure.messaging.eventhubs) for the respective client library.
https://azure.github.io/azure-sdk/java_documentation.html